crates/kind-openai/src/endpoints/chat.rs

```diff
@@ -199,6 +199,15 @@ impl<T> ChatCompletionChoice<T> {
 struct ChatCompletionResponseMessage<T> {
     content: T,
     refusal: Option<String>,
+    usage: Usage,
+}
+
+#[allow(dead_code)]
+#[derive(Deserialize)]
+struct Usage {
```

Comment:
Make this pub with accessors for each field, then re-export it from the lib.rs

Reply:
I made this pub and moved this to `lib.rs` and imported into `chat.rs`.